layouts/shortcodes/release-binaries.html

```diff
@@ -0,0 +1,93 @@
+<!-- Fetch release_binaries.json from kubernetes-sigs/downloadkubernetes to render in table -->
+{{ $response := getJSON "https://raw.githubusercontent.com/dipesh-rawat/downloadkubernetes/integrate-k8s-website/dist/release_binaries.json" }}
+
+{{ $Binaries := slice }}
+{{ $AllOSes := slice }}
+{{ $AllArch := slice }}
+{{ $AllVersions := slice }}
+
+{{ range $key, $value := $response }}
+    {{ if eq $key "Binaries" }}
+        {{ $Binaries = $value }}
+    {{ else if eq $key "AllOSes" }}
+        {{ $AllOSes = $value }}
+    {{ else if eq $key "AllArch" }}
+        {{ $AllArch = $value }}
+    {{ else if eq $key "AllVersions" }}
+        {{ $AllVersions = $value }}
+    {{ end }}
+{{ end }}
+
+<link href="{{ "css/release_binaries.css" | relURL }}" rel="stylesheet">
+<script defer src="{{ "js/release_binaries.js" | relURL }}"></script>
```

Comment:
Ideally, put this CSS link and extra script into the `head.html` partial. Look in `layouts/partials/head.html` for uses of `HasShortcode`: you can load a script to pages, but only when they use a particular shortcode.

Also make sure that this localizes OK. You might need to leave a note for the localization team(s) in the Markdown file where you use this shortcode.